packages/flutter/test/material/chip_test.dart

```diff
@@ -559,13 +559,15 @@ void main() {
     expect(calledDelete, isFalse);
     calledDelete = false;
 
+    final UniqueKey chipKey = UniqueKey();
     await tester.pumpWidget(
       _wrapForChip(
         child: Column(
           children: <Widget>[
             Chip(
+              key: chipKey,
```

Comment:
Why was this key needed?

Reply:
So I could print some values in the debug output that is no longer there. :-)

Removed.